dpnp/dpnp_iface_mathematical.py

```diff
@@ -1103,7 +1103,7 @@ def nancumsum(x1, **kwargs):
         if not isinstance(x1, dparray):
```

Comment:
All you external/third party tests fallbacks to numpy and doesn't check your implementation?
Please write more and more dpnp own tests for this functions.